src/db/dal/user.ts

```diff
@@ -1,24 +1,130 @@
+import AWS from 'aws-sdk';
 import createHttpError from 'http-errors';
 import { StatusCodes } from 'http-status-codes';
+import { Op, Order } from 'sequelize';
+import { uuid } from 'uuidv4';
 
+import { profileImageBucket } from '../../env';
 import { UserValidator } from '../../utils/userValidator';
 import UserModel, { IUserInput, IUserOuput } from '../models/User';
 
-const sanitizeInputPayload = (payload: IUserInput) => ({
-    ...payload,
-    id: undefined,
-    keycloak_id: undefined,
-    completed_registration: undefined,
-    creation_date: undefined,
-});
+let S3Client;
+try {
+    S3Client = new AWS.S3();
+} catch (error) {
+    //
```

Comment:
why not logging the error, at least?

Reply:
Pas de problème avec le try-catch. Je pense quand même qu'on devrait logger l'erreur.

Reply:
c'est justement ce que je souligne plus bas, je me demandais quel log y mettre. Car j'etais pas sur que ca throw dans le sdk si y'a pas de config. Et si c'est le cas c'est pas vraiment une err a log en console.err car c'est normal qu'on ai pas de config actuellement (sauf pour INCLUDE ou ya deja le s3 de set ici)
Mais je pense y mettre un console.warn ?